compile error in Parallel

<!--
If you have a question rather than reporting a bug please go to http://answers.opencv.org where you get much faster responses.
If you need further assistance please read [How To Contribute](https://github.com/opencv/opencv/wiki/How_to_contribute).

This is a template helping you to create an issue which can be processed as quickly as possible. This is the bug reporting section for the OpenCV library.
-->

##### System information (version)
<!-- Example
- OpenCV => 3.1
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2015
-->

- OpenCV => :grey_question:
- Operating System / Platform => :grey_question:
- Compiler => :grey_question:

##### Detailed description

<!-- your description -->

##### Steps to reproduce

<!-- to add code example fence it with triple backticks and optional file extension
    ```.cpp
    // C++ code example
    ```
 or attach as .txt or .zip file
-->

/home/naths/srcs/opencv/modules/core/src/parallel.cpp:442:19: error: use of
      undeclared identifier 'Range'
            pbody(Range(i, i + 1));
                  ^
1 error generated.
modules/core/CMakeFiles/opencv_core.dir/build.make:1268: recipe for target 'modules/core/CMakeFiles/opencv_core.dir/src/parallel.cpp.o' failed
make[2]: *** [modules/core/CMakeFiles/opencv_core.dir/src/parallel.cpp.o] Error 1

Latest pull from Master

--- a/modules/core/src/parallel.cpp
+++ b/modules/core/src/parallel.cpp
@@ -125,6 +125,8 @@
 #  define CV_PARALLEL_FRAMEWORK "pthreads"
 #endif
 
+using namespace cv;
+
 namespace cv
 {
     ParallelLoopBody::~ParallelLoopBody() {}

